docs/reference/cli-reference/cli-ref-sign.md

```diff
@@ -29,7 +29,12 @@ where `<package(s)>` is one or more `.nupkg` files.
 
 - **`-CertificateFingerprint`**
 
-  Specifies the SHA-1 fingerprint of the certificate used to search a local certificate store for the certificate.
+  Specifies the fingerprint of the certificate used to search a local certificate store for the certificate.
+
+  Starting with NuGet.exe 6.12, this option can be used to specify the SHA-1, SHA-256, SHA-384, or SHA-512 fingerprint of the certificate.
+  However, a `NU3043` warning is raised when a SHA-1 certificate fingerprint is used because it is no longer considered secure.
+
+  All the previous versions of the NuGet.exe continue to accept only SHA-1 certificate fingerprint.
```

Comment:
Consider putting this as a !NOTE to call it out more strongly.

Reply:
In https://github.com/NuGet/docs.microsoft.com-nuget/pull/3339#discussion_r1756275821, Andy pointed me to the following guidance from [docs' contribution guide](https://learn.microsoft.com/contribute/content/markdown-reference#alerts-note-tip-important-caution-warning).

> Avoid notes, tips, and important boxes. Readers tend to skip over them. It's better to put that info directly into the article text.

I am not sure if adding note block will have an impact on readers based on above guidance. Maybe we will merge this PR as is and update docs if needed in the future.